miden-tx/tests/integration/scripts/p2idr.rs

```diff
@@ -91,16 +91,15 @@ fn p2idr_script() {
     // --------------------------------------------------------------------------------------------
     // CONSTRUCT AND EXECUTE TX (Case "in time" - Target Account Execution Success)
     // --------------------------------------------------------------------------------------------
-    let data_store_1 = MockDataStore::with_existing(
-        Some(target_account.clone()),
-        Some(vec![note_in_time.clone()]),
-    );
+    let data_store_1 = TransactionContextBuilder::new(target_account.clone())
```

Comment:
nit: this should probably be named `tx_context`.

Reply:
done